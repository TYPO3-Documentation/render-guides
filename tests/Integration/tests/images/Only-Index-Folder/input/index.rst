=============
root document
=============


..  toctree::
    :glob:
    :hidden:

    */index
