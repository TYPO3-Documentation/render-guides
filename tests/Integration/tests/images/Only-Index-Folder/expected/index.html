<!-- content start -->
                <section class="section" id="root-document">
            <h1>root document<a class="headerlink" href="#root-document" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">Â¶</a></h1>

    </section>
        <!-- content end -->
